src/es/components/atoms/basket/Basket.js

```diff
@@ -7,33 +7,32 @@ import { Shadow } from '../../prototypes/Shadow.js'
  * @type {CustomElementConstructor}
  */
 export default class Basket extends Shadow() {
-
-  constructor(...args) {
+  constructor (...args) {
```

Comment:
I didn't changed any code in this file (Basket.js). 
The changes came from `npm run fix`